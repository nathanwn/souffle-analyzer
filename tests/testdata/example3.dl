.decl directed_edge(
    u: number,
    v: number
)

.decl path(
    u: number,
    v: number
)
