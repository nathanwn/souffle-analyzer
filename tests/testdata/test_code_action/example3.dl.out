example3.dl
0 | .decl directed_edge(
1 |     u: number,
2 |     v: number
3 | )
4 |
5 | .decl path(
6 |     u: number,
7 |     v: number
8 | )

=====

-- Cursor range --
example3.dl
0 | .decl directed_edge(
  | ~~~~~~~~~~~~~~~~~~~~
1 |     u: number,
  | ~~~~~~~~~~~~~~
2 |     v: number
  | ~~~~~~~~~~~~~
3 | )
  | ~
-- Insertion point --
example3.dl
0 | .decl directed_edge(
  | ^
--  Text  --
///
/// @attribute u
/// @attribute v
_____

-- Cursor range --
example3.dl
5 | .decl path(
  | ~~~~~~~~~~~
6 |     u: number,
  | ~~~~~~~~~~~~~~
7 |     v: number
  | ~~~~~~~~~~~~~
8 | )
  | ~
-- Insertion point --
example3.dl
5 | .decl path(
  | ^
--  Text  --
///
/// @attribute u
/// @attribute v
_____
